build-tools/automation/azure-pipelines.yaml

```diff
@@ -0,0 +1,214 @@
+# Xamarin.Android Pipeline
+
+trigger:
+  - master
+  - d16-*
+
+# Global variables
+variables:
+  BundleArtifactName: bundle
+  AutoProvisionArgs: /p:AutoProvision=True /p:AutoProvisionUsesSudo=True /p:IgnoreMaxMonoVersion=False
+
+stages:
+- stage: prepare
+  displayName: Prepare
+  variables:
+    MSBuildAbiArgs: >-
+      /p:AndroidSupportedTargetJitAbis=armeabi-v7a:arm64-v8a:x86:x86_64
+      /p:AndroidSupportedHostJitAbis=Darwin:mxe-Win32:mxe-Win64
+      /p:AndroidSupportedTargetAotAbis=armeabi-v7a:arm64:x86:x86_64:win-armeabi-v7a:win-arm64:win-x86:win-x86_64
+  jobs:
+  - job: create_bundle
+    displayName: Create Bundle
+    pool: Hosted Mac Internal
```

Comment:
Recommend setting the pool via a variable (if possible). Avoids having to commit a change to target a different pool

Reply:
I've set some variables in the job configuration page for this.